jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -272,3 +339,17 @@ def _get_partners_by_batch_type(cls, batch_type) -> List[CorpTypeModel]:
         corp_type_codes: List[str] = db.session.scalars(corp_type_query).all()
 
         return db.session.query(CorpTypeModel).filter(CorpTypeModel.code.in_(corp_type_codes)).all()
+
+    @classmethod
+    def _process_items(cls, items, ejv_header_model, sequence, link_type):
+        for item in items:
+            # Create Ejv file link and flush
```

Comment:
flush?

Reply:
it was outside the function, I moved it within the function now